packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -1193,6 +1193,166 @@ test('lib-wallet addOffer for autoswap swap', async (/** @type {LibWalletTestCon
   );
 });
 
+test('lib-wallet performAction suggestIssuer', async (/** @type {LibWalletTestContext} */ t) => {
+  t.plan(1);
```

Comment:
planning is helpful when the checks run in control structures. given the checks are all at top level of this function, this seems superfluous and means another thing to change when changing the test.

Reply:
Removed